ruby/ql/src/queries/analysis/Definitions.ql

```diff
@@ -65,17 +65,19 @@ newtype DefLoc =
     not exists(MethodBase m | m.getAChild+() = write)
   }
 
+pragma[noinline]
+ConstantWriteAccess definitionOf0(string fqn) {
+  fqn = resolveConstant(_) and
+  result =
+    min(ConstantWriteAccess w | w.getQualifiedName() = fqn | w order by w.getLocation().toString())
```

Comment:
Ordering by `toString` feels a bit brittle. Wouldn't it be better to order by path, startline, startcolumn, etc.?

Reply:
This bit was simply moved over from the old implementation.

Reply:
I have changed it.